packages/fxa-admin-panel/src/components/AdminLogs/index.tsx

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { useLazyQuery, gql } from '@apollo/client';
```

Comment:
Running `yarn workspace fxa-admin-panel build` produces these linter warnings.
```
./src/components/AdminLogs/index.tsx
  Line 5:17:  'useState' is defined but never used      @typescript-eslint/no-unused-vars
  Line 6:10:  'useLazyQuery' is defined but never used  @typescript-eslint/no-unused-vars
  Line 6:24:  'gql' is defined but never used           @typescript-eslint/no-unused-vars
```
We should delete the unused imports.

Reply:
Thanks @dannycoates ! I'll add these fixes